### DISCUSSION TITLE
OpenGL Deleting/Clearing/Removing Textures from memory?

### DISCUSSION THREAD

**Discussion by pnill** [ocornut/imgui#4095] [Old/Archived discussions]
So I'm using LoadTextureFromFile from the examples on handling textures in OpenGL but I don't know how to clear the textures from memory after I create them...

Not extremely familiar with OpenGL in general but tried, `glDeleteTextures` and that didn't seem to work I was still able to access them via my original pointer... I'm creating a sort of menu system and don't want the textures loaded into GPU memory if they're not shown on the screen currently.

Though I'm not sure if the GPU is going to care if there's say 100-200 textures loaded all at once either, wondering if anyone else has dealt with something similar.

**Comment by ocornut** [comment]
This topic has nothing to do with dear imgui itself, but may be better off asking/discussing on graphics programming or OpenGL forums.

**Reply by pnill** [reply]
@ocornut understood... was just thinking someone around here may have an answer and thought an 'discussion' would be less formal than creating an issue about it.
